### ISSUE TITLE
Could you provide an environment installation guidance? Or it's the same with MER2025Track1-3 vllm2?

### ISSUE THREAD

**Issue by Handsome-Fun** [cai-cong/MER25_personality#1]

**Comment by cai-cong** [comment]
Yes, you can run "pip install -r requirements.txt" now.

**Comment by Handsome-Fun** [comment]
> Yes, you can run "pip install -r requirements.txt" now.

Thks a lot! 

Besides, the labels of deception seems not being in https://huggingface.co/datasets/MDPEdataset/MER2025_personality/tree/main/deception; is it the same as the one in MDPE original dataset https://huggingface.co/datasets/MDPEdataset/MDPE_Dataset/tree/main/deception ?

**Comment by cai-cong** [comment]
> > Yes, you can run "pip install -r requirements.txt" now.
> 
> Thks a lot!
> 
> Besides, the labels of deception seems not being in https://huggingface.co/datasets/MDPEdataset/MER2025_personality/tree/main/deception; is it the same as the one in MDPE original dataset https://huggingface.co/datasets/MDPEdataset/MDPE_Dataset/tree/main/deception ?

Yes, the deception's labels of test data will not be released in this MER2025. You can use the labels of emotion to assist with personality prediction.

**Comment by Handsome-Fun** [comment]
> > > Yes, you can run "pip install -r requirements.txt" now.
> > 
> > 
> > Thks a lot!
> > Besides, the labels of deception seems not being in https://huggingface.co/datasets/MDPEdataset/MER2025_personality/tree/main/deception; is it the same as the one in MDPE original dataset https://huggingface.co/datasets/MDPEdataset/MDPE_Dataset/tree/main/deception ?
> 
> Yes, the deception's labels of test data will not be released in this MER2025. You can use the labels of emotion to assist with personality prediction.

Ok. When I'm running the training code, it output the following error: 
![Image](https://github.com/user-attachments/assets/5951b832-5304-4355-8560-a1726e863dbf)

I've found that the partition.csv has ids from 0 to 193 while the downloaded features start from 194. Is it the reason causing such error?

**Comment by cai-cong** [comment]
The training path should be modified to MDPE dataset path（https://huggingface.co/datasets/MDPEdataset/MDPE_Dataset） instead of the test set（https://huggingface.co/datasets/MDPEdataset/MER2025_personality）, as partitioning includes both the training and validation sets. The prediction of test set labels should run eval.py (using a trained model to predict labels)

**Comment by Handsome-Fun** [comment]
> The training path should be modified to MDPE dataset path（[https://huggingface.co/datasets/MDPEdataset/MDPE_Dataset）](https://huggingface.co/datasets/MDPEdataset/MDPE_Dataset%EF%BC%89) instead of the test set（[https://huggingface.co/datasets/MDPEdataset/MER2025_personality）](https://huggingface.co/datasets/MDPEdataset/MER2025_personality%EF%BC%89), as partitioning includes both the training and validation sets. The prediction of test set labels should run eval.py (using a trained model to predict labels)

Got it, thks a lot!
